### PR TITLE
Feat/Crypto Detail Page

### PR DESCRIPTION
**Pull request by zaunermax**
# Feature 🛠 

This PR includes the full implementation of the crypto currency asset detail page.

### CODE REVIEW

**Comment by zaunermax** [comment]
@zaunermax I think it's freaking cool to reply to a comment via Telegram chat!!

**Comment by zaunermax** [comment]
image

File: src/components/CoinDetails.vue [hunk 1]
```diff
@@ -1,21 +1,117 @@
...
+              id
+              circulatingSupply
+              totalSupply
+              priceUSD
+              priceBTC
+            }
+          }
+        `,
+        variables() {
+          console.log('coinid', this.$attrs.coinId)
```
Comment (mayrmax) [inline src/components/CoinDetails.vue:70]: remove console logs

File: src/components/CoinDetails.vue [hunk 2]
```diff
@@ -1,21 +1,117 @@
...
+          return { id: this.$attrs.coinId }
+        },
+        result(res) {
+          console.log('res', res)
```
Comment (mayrmax) [inline src/components/CoinDetails.vue:74]: same

File: src/components/TableView.vue [hunk 3]
```diff
@@ -27,78 +25,75 @@
...
+        `,
+        variables() {
+          return {
+            limit: this.perPage,
+            start: this.page * this.perPage,
+          }
+        },
+        fetchPolicy: 'cache-and-network',
+        result(result) {
+          console.log(result)
```
Comment (mayrmax) [inline src/components/TableView.vue:85]: remove console log

**Review by mayrmax — APPROVED** [review]

**Pull request merged**